src/apps/watchfaces/OswAppWatchfaceMonotimer.cpp

```diff
@@ -0,0 +1,146 @@
+
+#include "./apps/watchfaces/OswAppWatchfaceMonotimer.h"
+// #define GIF_BG
+
+#ifdef ANIMATION
+#include <anim_matrix.h>
+#endif
+
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_config.h>
+#include <osw_config_keys.h>
+#include <osw_hal.h>
+
+#ifdef GIF_BG
+#include "./apps/_experiments/gif_player.h"
+#endif
+
+void OswAppWatchfaceMonotimer::drawNShiftedTicks(Graphics2D *gfx, uint8_t cx, uint8_t cy, uint8_t r1, uint8_t r2, uint8_t nTicks, float shift, uint16_t color) {
+    const float deltaAngle = 360.0 / nTicks;
+    for(uint16_t i=0; i<nTicks; ++i) {
+        gfx->drawTick(cx, cy, r1, r2, (i * deltaAngle) + shift, color);
+    }
+}
+
+void OswAppWatchfaceMonotimer::drawNShiftedMaskedTicks(Graphics2D *gfx, uint8_t cx, uint8_t cy, uint8_t r1, uint8_t r2, uint8_t nTicks, float shift, uint16_t m, uint16_t color) {
```

Comment:
I think it would be good to write an explanation about the factors that go into the function.
```
/*
@param sample blah blah
*/
void OswAppWatchfaceMonotimer::drawNShiftedMaskedTicks()...
```
What kind of functions does this function?

Reply:
The drawNTicks always draws the first Tick at 0°. Since I need to draw many ticks of differnent sizes, drawing more ticks than needed and overdrawing ticks is not efficient. Thus, I've added modified functions for this.

I've now added explanations.